tsconfig-project.json

```diff
@@ -4,7 +4,7 @@
     "outDir": "./build"
   },
   "include": [
-    "src",
+    "src", "test",
```

Comment:
I tried this. It overwrites the sources if they are named the same way.

Reply:
Yes, I've already removed it but didn't commit ( I used it to opimize some tests)